lib/fathead/mdn_css/parse.pl

```diff
@@ -296,7 +296,7 @@ sub create_redirects {
 
     # Capture content inside parentheses
     # E.g. "::before (:before)"
-    if ( $title =~ m/(::.+) \((:.+)\)/ ) {
+    if ( $title =~ m/(.+) \((:.+|-webkit.+)\)/ ) {
```

Comment:
@rasikapohankar yes, because we're redirecting the same query to two different queries, which doesn't work. Redirects must be 1-to-1 and unique. Any query that acts as a redirect cannot redirect to more than one article. Likewise if a query is specified as a redirect, it can't be used for anything else (like an article or a disambiguation).

Seeing as `:-moz-placeholder` and `::-moz-placeholder` are two different articles, and we can't determine which the user might want to see for the query `-moz-placeholder`, we should show a disambiguation result.

If there is only 1 of these cases then we may want to create a special case for this, otherwise, we should find a programmatic way of detecting these. We could even prepared a `disambigations.yaml` file that lists out all the disambiguations:

The LHS for every line in a Fathead output.txt must be unique. Any duplicates are technically overridden internally, but we shouldn't be generating any.